composer.json

```diff
@@ -5,25 +5,22 @@
     "keywords": ["framework", "laravel"],
     "license": "MIT",
     "require": {
-        "php": "^8.0.2",
-        "ext-exif": "*",
-        "ext-gd": "*",
-        "fakerphp/faker": "^1.9.1",
-        "fruitcake/laravel-cors": "^2.0",
-        "guzzlehttp/guzzle": "^7.0.1",
-        "inertiajs/inertia-laravel": "^0.6.4",
-        "laravel/framework": "^8.65",
-        "laravel/sanctum": "^2.11",
-        "laravel/tinker": "^2.5",
-        "league/glide-laravel": "^1.0"
+        "php": "^8.1",
+        "fruitcake/laravel-cors": "dev-develop",
```

Comment:
fruitcake/laravel-cors is not in Laravel 10. It was removed in Laravel 9.

Here is the PR that made the change:

https://github.com/laravel/laravel/pull/5825